hooks/boot/04-services.rb

```diff
@@ -0,0 +1,36 @@
+require 'open3'
+
+module ServicesHookContextExtension
+  ALL_POSSIBLE_SERVICES = [
+    'httpd.service', 'apache.service', # RH vs Debian
+    'postgresql*',
+    'foreman*',
+    'dynflow*',
+    '*mongo*',
+    '*redis*',
+    'pulp*',
+  ]
+
+  def start_services(services)
+    raise "Services must be specified" if services.empty?
+
+    logger.debug("Starting services #{services.join(', ')}")
+    stdout_and_stderr_str, status = Open3.capture2e('systemctl', 'start', *services)
+    fail_and_exit("Failed to start services: #{stdout_and_stderr_str}", status.exitstatus) unless status.success?
```

Comment:
Whats the reason behind the combined stdout and stderr here? (vs just putting stderr out)

Reply:
I wasn't quite sure about the output. just grabbing the entire output allows to debug was my reasoning. Didn't think it through that much.

Reply:
Suppose its a starting place.